gallery/src/components/NxSmallVulnerabilityCounter/NxSmallVulnerabilityCounterPage.tsx

```diff
@@ -0,0 +1,180 @@
+            </NxTable.Cell>
+          </NxTable.Row>
+          <NxTable.Row>
+            <NxTable.Cell>maxDigits</NxTable.Cell>
+            <NxTable.Cell>number</NxTable.Cell>
+            <NxTable.Cell>No</NxTable.Cell>
+            <NxTable.Cell>3</NxTable.Cell>
+            <NxTable.Cell>
+              <NxP>
+                The maximum number of digits to display within the counter. This value is used to make all counters
+                across an entire table consistent in width. Each count is given space for this number of digits along
+                with a plus sign for rendering values that go over the maximum.  For instance, if this prop is set to
+                2, then the counters will display only one or two digits numbers. If any count in that case is greater
+                than two digits, the counter will display "99+". The counters will be exactly wide enough to display
+                "99+" regardless of their actual content. Note that since the default for this prop is 3, by default
+                counters will be wide enough to display "999+".
+              </NxP>
+              <NxP>
+                If <NxCode>maxDigits</NxCode> is set to <NxCode>Infinity</NxCode>, then a different behavior is
+                activated: all counters within this instance of <NxCode>NxSmallVulnerabilityCounter</NxCode> will be as
+                wide as needed to display the largest number in any one of them. That is, within the component instance,
+                all of the counters will be the same width, and that width will depend on the data present. However,
+                when compared to another instance that uses different data, the widths may not be the same.
+              </NxP>
+            </NxTable.Cell>
+          </NxTable.Row>
+          <NxTable.Row>
+            <NxTable.Cell>HTML <NxCode>&lt;div&gt;</NxCode> Attributes</NxTable.Cell>
+            <NxTable.Cell>
+              <NxTextLink external href="https://developer.mozilla.org/en/docs/Web/HTML/Element/div">
+                HTML div Attributes
+              </NxTextLink>
+            </NxTable.Cell>
+            <NxTable.Cell>No</NxTable.Cell>
+            <NxTable.Cell />
+            <NxTable.Cell>
+              <NxCode>NxSmallVulnerabilityCounter</NxCode> supports any HTML attribute that's normally
+              supported by <NxCode>&lt;div&gt;</NxCode> elements.
+            </NxTable.Cell>
+          </NxTable.Row>
+        </NxTable.Body>
+      </NxTable>
+    </GalleryDescriptionTile>
+
+    <GalleryExampleTile title="NxSmallVulnerabilityCounter Table Example"
+                        id="nx-small-vulnerability-counter-example"
+                        liveExample={NxSmallVulnerabilityCounterExample}
+                        codeExamples={NxSmallVulnerabilityCounterCode}>
+      A table containing an <NxCode>NxSmallVulnerabilityCounter</NxCode> in each row.
+      Note that tables are the only intended use case for this component.
```

Comment:
I don't think this is true.  I think this verbiage on the `NxSmallThreatCount` page pre-dates the addition of the `maxDigits=Infinity` behavior
```suggestion
```